autobuild/linux/autobuild_deb_1_prepare.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh -e
+
+# autobuild_1_prepare: set up environment, install Qt & dependencies
+
+
+###################
+###  PROCEDURE  ###
+###################
+
+# Sets up the environment for autobuild on Linux
+
+echo "Update system..."
+sudo apt-get -qq update
+# We don't upgrade the packages. If this is needed, just uncomment this line
```

Comment:
No, just remove these two commented lines.  We need apt updated and then the install should pull the required packages correctly.

Reply:
can do, no strong own opinion on this